hack/annotate.sh

```diff
@@ -6,6 +6,9 @@ set -o pipefail
 
 base="$( dirname "${BASH_SOURCE[0]}" )/.."
 
+# The following is an array of all the 4.x releases that have ReleaseConfig definitions, sorted by version from Highest to Lowest:
+# [4.11, 4.10, 4.9, 4.8, ...]
+releases=( $(ls "${base}/core-services/release-controller/_releases/" | grep -Eo "4\.[0-9]+" | sort -Vr | uniq) )
```

Comment:
Consider limiting it to ocp, just in case.
```suggestion
releases=( $(ls "${base}/core-services/release-controller/_releases/" | grep "ocp-" | grep -Eo "4\.[0-9]+" | sort -Vr | uniq) )
```

Reply:
Done!